terraform/etl/53-aws-glue-housing-extract-mmh-vulnerabilities-to-refined.tf

```diff
@@ -24,17 +24,16 @@ module "housing_mtfh_case_notes_enriched_to_refined" {
     "--enable-glue-datacatalog"          = "true"
     "--enable-continuous-cloudwatch-log" = "true"
     "--additional-python-modules"        = "spacy==3.7.4"
-    "--s3_source_mtfh_notes"             = module.department_housing_data_source.raw_zone_catalog_database_name
+    "--s3_source_mtfh_notes"             = "s3://dataplatform-stg-raw-zone/housing/"
     "--source_table_mtfh_notes"          = "mtfh_notes"
-    "--s3_source_tenure"                 = module.department_housing_data_source.refined_zone_catalog_database_name
+    "--s3_source_tenure"                 = "s3://dataplatform-stg-refined-zone/housing/mtfh/"
```

Comment:
You are using a staging S3 bucket. We need this work in staging and prod, so need use below:
`s3://${module.raw_zone_data_source.bucket_id}/house/`
`s3://${module.refined_zone_data_source.bucket_id}/housing/mtfh`